dynamic_analysis_pkg/dynamic_analysis_pkg/__init__.py

```diff
@@ -0,0 +1,167 @@
+            writable_content = writable_content.replace(self.ORIGIN_DIR_PATH, self.local_dir)
+
+            with open(file_output_path, 'w') as fd:
+                fd.write(writable_content)
+
+        else:
+            x.download(str(file_name)) # moves to home always
+            move(file_name, file_output_path)
+
+        print('- [{}] downloaded'.format(file_name))
+
+    def download_blueconfig(self, file_name='BlueConfig'):
+        self.download_file_to_storage(file_name)
+
+    def download_report(self):
+        reports = [x for x in self.files_list if '.bbp' in x]
+        if not reports:
+            print('No reports were found')
+            return
+        for report in reports:
+            self.download_file_to_storage(report)
+
+    def download_out_dat(self):
+        self.download_file_to_storage('out.dat')
+
+
+class FetchMultipleResults:
+
+    values = []
+
+    def __init__(self, token, sim_list_ids, base_working_directory=None):
+
+        total_sims = len(sim_list_ids)
+        for idx, sim in enumerate(sim_list_ids):
+            print('({}/{})'.format(idx + 1, total_sims))
+            self.values.append(Results(token, sim, base_working_directory))
+
+
+class JobFinder:
+
+    jobs = []
+    sorted_jobs = None
+
+    def __init__(self, token):
+        tr = unicore_client.Transport(token)
+        client = unicore_client.Client(tr, UNICORE_ENDPOINT)
+        self.jobs = client.get_jobs()
+
+    def show_list(self, amount_jobs_to_show=None):
+        full_jobs_list = []
+        if amount_jobs_to_show is None:
+            amount_jobs_to_show = 1000
+
+        for index, job in enumerate(self.jobs[:amount_jobs_to_show]):
+            raw_time = job.properties['submissionTime']
+            full_time = datetime.strptime(raw_time, '%Y-%m-%dT%H:%M:%S%z')
+            date = full_time.strftime('%d-%m-%Y %H:%M')
+            name = job.properties['name']
+            job_id = basename(job.properties['_links']['self']['href'])
+            full_jobs_list.append([index, name, date, job_id])
+            print('Fetching jobs ({}/{})'.format(index + 1, amount_jobs_to_show or len(jobs) + 1), end = '\r')
+
+        columns = ['index', 'name', 'date', 'job_id']
+        self.sorted_results = pd.DataFrame(full_jobs_list, columns=columns).sort_values(by=['date'])
+
+        pd.options.display.max_columns = None
+        pd.options.display.width=None
+
+        print('\n') # to stop the \r
+        showing_columns = ['index', 'name', 'date']
+        # insert index for user selection
+        self.sorted_results['index'] = range(0, amount_jobs_to_show)
+        print(tabulate(self.sorted_results[showing_columns], headers=showing_columns, showindex=False))
+
+
+    def find_id_by_index(self):
+        job_selection_index = input('Please select the index that you want to import:')
+        try:
+            selected_index = int(job_selection_index)
+        except:
+            print('ERROR: enter correct index')
```

Comment:
You want to raise here, no ?

```suggestion
            raise ValueError('ERROR: enter correct index')
```
?